packages/manager/modules/account-sidebar/src/payment-mean/template.html

```diff
@@ -1,4 +1,4 @@
-<div class="manager-hub-payment-mean">
+<div class="manager-hub-payment-mean" data-navi-id="paymentMean-block">
```

Comment:
How does it work ?

Reply:
The attributes are given by QA team and are used for end to end tests

Reply:
Is there a documentation about this ?

This looks like an extra unrequired overload, so could easily be fixed by coding semantically once again

Reply:
I think we shoudn't need this for targeting element in tests (some great tools are here to help us to target elements as the customer target them: using text and global type (object, input...) like [Testing Library](https://testing-library.com/))
And IMO, this `navi-id` is ok in the context of the hub, but if you are in a global payment mean section, is `paymentMean-block` ok (because you could have more than one block in paymentMean page for example)? I think we should be more precise, here it's a `paymentMean-block` in a user sidebar.

Reply:
cc @fchabrolle

Reply:
i think it's a block for payments inside the sidebar, if we can have multiple payments listed, there is still only one payment block containging them all. text shouldn't be used (it can be translated for example) theses attributes are here to find inputs easily

Reply:
Could be handled right after: MANAGER-4630.

thanks all for your feedback